angular/core/components/attachment_preview/attachment_preview.coffee

```diff
@@ -12,4 +12,4 @@ angular.module('loomioApp').directive 'attachmentPreview', ->
       $scope.attachment[$scope.mode]
 
     $scope.displayMode = ->
-      _.contains ['thread', 'context'], $scope.mode
+      _.contains ['thread', 'context', 'proposal'], $scope.mode
```

Comment:
I'd picked the 'context' word with a mind that we'd be able to reuse it for proposals (maybe there's a better name for it?) Anyway I don't think we should need to have a whole new mode for this.